modules/storages/app/models/storages/nextcloud_storage.rb

```diff
@@ -0,0 +1,11 @@
+class Storages::NextcloudStorage < Storages::Storage
+  store :provider_fields, accessors: %i[username password has_managed_project_folders]
+
+  def has_managed_project_folders=(value)
+    super(!!value)
+  end
+
+  def has_managed_project_folders
```

Comment:
@ba1ash Even rubocop wants this method to be a predicate with a question mark at the end :-)

Reply:
True :)
I added an alias to the method.